### DISCUSSION TITLE
Problem with the Bibliography

### DISCUSSION THREAD

**Discussion by anab007** [joaomlourenco/novathesis#480] [Q&A] [answered]
Hello,
I'm doing my thesis on texmaker since Overleaf doesn't compile the template. In the file bib I put my bibliography but the Bibliography doesn't appear on the template at all. In attachament, I send a print of my bib file. My version of the template is 7.1.18 and also I want to emphasise that I didn't do any changes to the bib definitions.

Thank you in advance for your help and if it could be quick, as I need to submit a provisional version of my thesis tomorrow.
<img width="602" alt="bib print 1" src="https://github.com/joaomlourenco/novathesis/assets/156536766/74c033ba-10ee-48a1-8c81-a5c3935cdf6b">
<img width="598" alt="bib print 2" src="https://github.com/joaomlourenco/novathesis/assets/156536766/8a10b318-69db-48df-9d16-cf9555fc7bd7">
<img width="591" alt="bib print 3" src="https://github.com/joaomlourenco/novathesis/assets/156536766/f5c66758-806b-4983-becc-33a437e941f7">

<img width="589" alt="bib print 4" src="https://github.com/joaomlourenco/novathesis/assets/156536766/c3ac06c4-8719-4834-b961-bf1245dd3cb0">

**Comment by joaomlourenco** [comment, marked as answer]
You have to configure the editor to use Bieber and not biblater. 

Even better, configure it to use "latexmk"!

**Reply by anab007** [reply]
I had already configured the editor to use Bieber. With regard to "latexmk", I have this in your configuration (attached is an image of my configuration), should I add anything specific?

Thanks again for your help!
<img width="714" alt="config texmaker" src="https://github.com/joaomlourenco/novathesis/assets/156536766/3d466491-3002-40ac-949a-7d8d601becfd">

**Reply by joaomlourenco** [reply]
That's the configuration of the commands. 
Go to the next "tab" (compiler) and select latexmk as the compiler.

**Reply by anab007** [reply]
Thank you very much for solving my problem!
